install/kubernetes/cilium/values.yaml

```diff
@@ -1076,6 +1076,12 @@ ipam:
 # -- Configure the eBPF-based ip-masq-agent
 ipMasqAgent:
   enabled: false
+  config:
```

Comment:
After #20066 was merged, now these changes need to be made to `install/kubernetes/cilium/values.yaml.tmpl` instead, then run `make -C install/kubernetes` to update this file.

Reply:
Okay